web/styles/subscriptions.css

```diff
@@ -551,6 +561,20 @@ h4.user_group_setting_subsection_title {
     width: 100%;
 }
 
+.left .group-name .fa-ban,
+.right .group-name .fa-ban {
```

Comment:
Icon does not look perfectly aligned vertically with the name.

Reply:
It will depend on group names -

when using `align-items: center` -
![Screenshot from 2025-02-11 09-21-58](https://github.com/user-attachments/assets/19e060b1-6b16-48b3-95a5-0377875c962a)

Current version -
![Screenshot from 2025-02-11 09-22-35](https://github.com/user-attachments/assets/b7aab7a9-b311-443f-a1b3-15d56951c996)